pkg/infra/usagestats/statscollector/service.go

```diff
@@ -161,6 +161,10 @@ func (s *Service) collectSystemStats(ctx context.Context) (map[string]interface{
 	m["stats.active_data_keys.count"] = statsResult.ActiveDataKeys
 	m["stats.public_dashboards.count"] = statsResult.PublicDashboards
 	m["stats.correlations.count"] = statsResult.Correlations
+	m["stats.database.created.time"] = statsResult.DatabaseCreatedTime
```

Comment:
@DanCech is there a better way to report this metric?   maybe age?

Reply:
I think created time as a timestamp is fine, I also switched up the driver to just report the string directly